content_schemas/dist/formats/world_index/frontend/schema.json

```diff
@@ -307,23 +307,45 @@
           "items": {
             "type": "object",
             "required": [
+              "active",
+              "content_id",
               "name",
               "slug",
-              "active"
+              "updated_at"
             ],
             "additionalProperties": false,
             "properties": {
               "active": {
                 "description": "Whether the location is currently active",
                 "type": "boolean"
               },
+              "analytics_identifier": {
+                "description": "The analytics identifier for the international delegation",
```

Comment:
Assuming this is `international delegation` on purpose and shouldn't simply be `world location`?

Reply:
Looks like I went a bit over the top with find-and-replace.  These should be labelled correctly for each section.

I'll update.

Reply:
Ah, that makes sense! I was struggling to see how a world location was the same as an international delegation

Reply:
I left this comment in the wrong place which doesn't help. 

This looks good now since your change, the only mentions of "international delegation" are within the links to the related international delegations